docs/specs/D-kit/Auto_Driving--local_map_generation_cn.md

```diff
@@ -1,7 +1,5 @@
 ﻿# 封闭园区自动驾驶搭建--定位地图制作
-
-多传感器融合定位(msf)，结合了GNSS、IMU和激光雷达点云定位，可以提高定位精度，而激光雷达点云定位，则需要提前制作定位地图(local_map),本文将说明开发者套件的定位地图制作方法。
-
+多传感器融合定位(msf)，结合了GNSS、 IMU和激光雷达点云信息来定位，可以提高定位精度，而激光雷达点云定位，则需要提前制作定位地图(local_map),本文将说明开发者套件的定位地图制作方法。
```

Comment:
Surround headers by a single empty line except at the beginning of the file.

https://cirosantilli.com/markdown-style-guide

Reply:
I have Modified it already, please review it again. Thank you!